packages/node_modules/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1026,8 +1018,11 @@ export const VIDEO_RESOLUTIONS = {
 export const MAX_FRAMESIZES = {
   [QUALITY_LEVELS.LOW]: 1620,
   [QUALITY_LEVELS.MEDIUM]: 3600,
-  [QUALITY_LEVELS.HIGH]: 8704,
-  [QUALITY_LEVELS.FULL_HD]: 36864
+  [QUALITY_LEVELS.HIGH]: 8192,
```

Comment:
We checked the resolution table on wiki - https://en.wikipedia.org/wiki/Advanced_Video_Coding
we have updated max frame size value (@30 fps) for 480p, 720p and 1080p, but we didn't find reference for 360p in wiki.